mitmproxy/test/tflow.py

```diff
@@ -115,6 +115,26 @@ def tflow(client_conn=True, server_conn=True, req=True, resp=None, err=None):
     return f
 
 
+def tdummyflow(client_conn=True, server_conn=True, err=None):
+    class DummyFlow(flow.Flow):
```

Comment:
Yes, let's move it out.
This creates a new class everytime you call the function, that's a bit weird.